postcss.config.js

```diff
@@ -21,5 +21,6 @@ module.exports = {
         "focus-visible-pseudo-class": false,
       },
     },
+    "postcss-discard-comments": {},
```

Comment:
👍

Reply:
Not required but could be a nice addition. I couldn't think of a reason to want to leave comments in the output CSS files.